dotCMS/src/main/java/com/dotcms/rest/BaseRestPortlet.java

```diff
@@ -141,7 +140,7 @@ private String getJspResponse ( HttpServletRequest request, HttpServletResponse
 	@Path("/layout/{params:.*}")
 	@Produces("text/html")
 	public Response getLayout ( @Context HttpServletRequest request, @Context HttpServletResponse response, @PathParam( "params" ) String params )
-			throws DotDataException, DotSecurityException, ServletException, IOException, DotRuntimeException,
+			throws DotDataException, ServletException, IOException, DotRuntimeException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.exception.DotRuntimeException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)